firefox-mobile/manifest.json

```diff
@@ -26,5 +26,6 @@
   },
   "browser_action": {
     "default_title": "Report site issue"
-  }
+  },
+  "default_locale": "en"
```

Comment:
Hmm, I totally don't remember if this is meaningful or a mistake, but why doesn't firefox-mobile use the shared manifest? Maybe worth a follow up bug to investigate and merge.

Reply:
There's a separate manifest file source for firefox-mobile with 2 differences from the shared -- no context menu permissions & no default icons in `browser_action`. Not sure if that's actually necessary (e.g., fennec would freak out if they were included rather than just ignoring them), but I can check on that.